### DISCUSSION TITLE
How to make the worker start with the last jobs when starting the app?

### DISCUSSION THREAD

**Discussion by ionutpopa** [taskforcesh/bullmq#2760] [Q&A]
Hello, I have an app that runs some jobs in an interval of time.
There is a scenario where the app closes and starts again but when the app starts it doesn't process the last jobs it had until a very long amount of time passes (sometimes 5 minutes sometimes 6 minutes).
The strange thing is that if I am checking my Redis Stack I see that all the jobs that didn't got to process first time are in the active state and this makes me wonder: Why the heck those jobs are in the active state but won't start processing faster?

Does this happened to somebody as well or do I have bad luck?

I do use a backoff strategy of exponential with a delay of 1000 but I tested without the backoff strategy and I have the same result.

Please let me know if you can help me with this issue!

**Comment by manast** [comment]
It seems like you are closing the app while workers are processing jobs, which leads this jobs to become stalled. The default time for stalled jobs is 30 seconds, so that's what it should take to start processing again: https://docs.bullmq.io/guide/jobs/stalled

**Reply by manast** [reply]
This may also be relevant to your case: https://docs.bullmq.io/guide/going-to-production#gracefully-shut-down-workers
